server/test/realtime/webhook_connector_test.exs

```diff
@@ -1,62 +1,149 @@
 defmodule Realtime.WebhookConnectorTest do
   use ExUnit.Case
 
+  import Mock
+
   alias Realtime.Adapters.Changes.Transaction
-  alias Realtime.Configuration
-  alias Realtime.TransactionFilter.Filter
+  alias Realtime.Configuration.{Webhook, WebhookEndpoint}
+  alias Realtime.Decoder.Messages.Relation.Column
   alias Realtime.WebhookConnector
 
-  test "notify calls notification function after filtering the handlers" do
-    {:ok, call_counter} = Agent.start_link fn -> 0 end
-    notify = fn (_, _) ->
-      Agent.update(call_counter, fn c -> c + 1 end)
-      Task.async(fn ->
-	{:ok, %{status_code: 200}}
-      end)
+  @test_endpoint "https://webhooktest.site"
+  @type_modifier 4_294_967_295
+  @txn %Transaction{
+    changes: [
+      %Realtime.Adapters.Changes.NewRecord{
+        columns: [
+          %Column{
+            flags: [:key],
+            name: "id",
+            type: "int8",
+            type_modifier: @type_modifier
+          },
+          %Column{
+            flags: [],
+            name: "details",
+            type: "text",
+            type_modifier: @type_modifier
+          },
+          %Column{
+            flags: [],
+            name: "user_id",
+            type: "int8",
+            type_modifier: @type_modifier
+          }
+        ],
+        commit_timestamp: %DateTime{
+          calendar: Calendar.ISO,
+          day: 22,
+          hour: 05,
+          microsecond: {0, 0},
+          minute: 22,
+          month: 2,
+          second: 19,
+          std_offset: 0,
+          time_zone: "Etc/UTC",
+          utc_offset: 0,
+          year: 2021,
+          zone_abbr: "UTC"
+        },
+        record: %{
+          "details" =>
+            "The SCSI system is down, program the haptic microchip so we can back up the SAS circuit!",
```

Comment:
:)

Reply:
@kiwicopple I wish I came up with it! All @fracek!